backend/src/map_model.rs

```diff
@@ -229,16 +229,14 @@ impl MapModel {
         self.roads
             .iter()
             .min_by_key(|r| {
-                let diff1 = Line::new(
+                let diff1 = Euclidean.distance(
```

Comment:
This was just a matter of taste - measuring line.length is the same as the distance between two points.